spec/emerald_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'spec_helper'
+require 'tempfile'
+require 'helper/user_comms'
+include Emerald
+require 'emerald/parser'
+require 'emerald/compiler'
+
```

Comment:
I'm def not asking you to change it though, just expressing my personal taste. Other preferences are equally valid.